README.md

```diff
@@ -1235,6 +1233,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Sam Sharma](https://github.com/javascript-spec)
   - [Saminda P](https://github.com/zaminda)
   - [Samuel Z. Bouye](https://github.com/zamblebi)
```

Comment:
Your name should be before [Samuel Z. Bouye] [L1235], kindly add your name before him without changing anything else to satisfy the alphabetical order requirement to get your PR merged with no conflicts

Reply:
Done , thanks guys